PrebidMobile/AdUnits/BannerBaseAdUnit.swift

```diff
@@ -12,8 +12,14 @@ limitations under the License.
 
 import Foundation
 
+@objc public protocol BannerBasedAdUnitProtocol {
+    var bannerParameters: BannerParameters { get set }
+}
+
+@available(*, deprecated, message: "This class is deprecated.")
```

Comment:
We should inform publishers which class they should use instead of the deprecated one.